_1st_row:
  '-': !!python/tuple
  - 390
  - 322
  '0': !!python/tuple
  - 373
  - 322
  '1': !!python/tuple
  - 225
  - 322
  '2': !!python/tuple
  - 244
  - 322
  '3': !!python/tuple
  - 264
  - 322
  '4': !!python/tuple
  - 279
  - 322
  '5': !!python/tuple
  - 297
  - 322
  '6': !!python/tuple
  - 306
  - 322
  '7': !!python/tuple
  - 322
  - 322
  '8': !!python/tuple
  - 339
  - 322
  '9': !!python/tuple
  - 356
  - 322
  '=': !!python/tuple
  - 407
  - 322
  "\b": !!python/tuple
  - 424
  - 322
_2nd_row:
  '[': !!python/tuple
  - 394
  - 339
  ']': !!python/tuple
  - 411
  - 339
  e: !!python/tuple
  - 270
  - 337
  i: !!python/tuple
  - 344
  - 339
  o: !!python/tuple
  - 359
  - 339
  p: !!python/tuple
  - 376
  - 339
  q: !!python/tuple
  - 235
  - 337
  r: !!python/tuple
  - 287
  - 337
  t: !!python/tuple
  - 304
  - 337
  u: !!python/tuple
  - 325
  - 339
  w: !!python/tuple
  - 252
  - 337
  y: !!python/tuple
  - 309
  - 339
_3rd_row:
  '''': !!python/tuple
  - 399
  - 355
  "\n": !!python/tuple
  - 422
  - 355
  ;: !!python/tuple
  - 381
  - 355
  a: !!python/tuple
  - 237
  - 354
  d: !!python/tuple
  - 273
  - 354
  f: !!python/tuple
  - 290
  - 354
  g: !!python/tuple
  - 308
  - 354
  h: !!python/tuple
  - 311
  - 355
  j: !!python/tuple
  - 328
  - 355
  k: !!python/tuple
  - 345
  - 355
  l: !!python/tuple
  - 364
  - 355
  s: !!python/tuple
  - 256
  - 354
_4th_row:
  ',': !!python/tuple
  - 353
  - 370
  .: !!python/tuple
  - 372
  - 370
  /: !!python/tuple
  - 389
  - 370
  b: !!python/tuple
  - 317
  - 368
  c: !!python/tuple
  - 281
  - 368
  m: !!python/tuple
  - 336
  - 370
  n: !!python/tuple
  - 319
  - 370
  v: !!python/tuple
  - 299
  - 368
  x: !!python/tuple
  - 263
  - 368
  z: !!python/tuple
  - 245
  - 368
_5th_row:
  ' ': !!python/tuple
  - 317
  - 385
aruco_ref:
  angle: 0.0 #1.04
  cx: 543
  cy: 286
